cypress/integration/focus-trap-demo.spec.js

```diff
@@ -67,6 +67,114 @@ describe('<FocusTrap> component', () => {
     });
   });
 
+  describe('demo: animated-dialog', () => {
```

Comment:
Animated demo tests are just a copy-paste of the default test.

They don't test for a console log message after `onPostActivate` and `onPostDeactivate` are called.

Reply:
In order to increase the code coverage I might need to make the tests slightly different 😕
Though there is definitely some refactoring that can be done here. I was hoping to not have to put to much effort into test writing though.

Or do you think the coverage is enough as is? You can see the code coverage warnings in the PR.

I should probably add tests for checking that the source code links go to the right place.

Reply:
I know, I loathe tests. 😄 But they help give us confidence. It would be good to at least cover the fact that the new options are being called as expected. That should be fairly easy to do as unit tests in https://github.com/focus-trap/focus-trap-react/blob/master/test/focus-trap-react.test.js I don't think the Cypress tests need to add more coverage.

> I should probably add tests for checking that the source code links go to the right place.

If you mean the "view source" links, I don't think that's necessary. It's more important to spend time on testing actual library functionality.

Reply:
Ok, I've added unit tests now and made sure that they fail if the options are not behaving correctly.

I didn't add unit tests to core focus-trap though when I added the new options. You might want to take the test I wrote and apply them to core focus-trap.

Any reason you don't have code coverage set up for core focus-trap?

Reply:
> Any reason you don't have code coverage set up for core focus-trap?

Because the person who set it up for this repo didn't for that one. 🙂 🤷‍♂️ More of a nice-to-have for me right now.